### PR TITLE
[TensorExpr] Add lowerings for quantized ops: cat, mul, conv1d, relu.

### PR DESCRIPTION
**Pull request by ZolotukhinM**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #66522
* #66847
* #66515
* #66514
* __->__ #69055
* #68857
* #68856
* #68855
* #68854

Differential Revision: [D32710325](https://our.internmc.facebook.com/intern/diff/D32710325)

### CODE REVIEW

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32710325).

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32710325).

File: torch/csrc/jit/tensorexpr/external_functions.cpp [hunk 1]
```diff
@@ -479,6 +506,34 @@ void nnc_aten_quantized_mul_scalar(
...
+  const c10::ScalarType a_qdtype = static_cast<c10::ScalarType>(extra_args[2]);
+  auto qa = from_blob_quantized(
+      buf_data[1],
+      // NOLINTNEXTLINE
+      tensors[1].sizes(),
+      tensors[1].strides(),
+      a_qscale,
+      a_qzero,
+      toQIntType(a_qdtype));
+  auto r = at::relu(qa);
```
Comment (navahgar) [inline torch/csrc/jit/tensorexpr/external_functions.cpp:533]: Should this be calling `quantized_relu` function instead? (like the `nnc_aten_quantized_cat` case below)

Comment (ZolotukhinM) [reply torch/csrc/jit/tensorexpr/external_functions.cpp:533]: I'm not sure, but it does produce correct results :) I suspect it dispatches to `quantized::relu` under the hood when `qa` is a QTensor.

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32710325).

**Review by navahgar — APPROVED** [review]
LGTM

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32710325).

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32710325).

**Comment by ZolotukhinM** [comment]
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32710325).

**Pull request closed without merging**